lib/application.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Application
+ * @constructor
+ * @param {Object} client Catapult client
+ */
+
+var Application = function (client) {
+	/**
+	 * List the user's applications
+	 * @param {Object} params Parameters for filtering applications
+	 * @param {Number} [params.page] The specified page requested when querying
```

Comment:
@dtolb Could I get some clarification for this? On http://ap.bandwidth.com/docs/rest-api/applications/ it says page is a supported parameter